user-console/src/main/resources/org/pentaho/mantle/public/browser/js/browser.js

```diff
@@ -339,55 +358,69 @@ define([
     },
 
     updateFolderButtons: function( _folderPath) {
+      let isRepoPath;
+      if (_folderPath){
+        isRepoPath = isRepositoryPath(_folderPath);
+      } else {
+        isRepoPath = false;
+      }
```

Comment:
No need to change, but for the future, consider something more JSy, concise and still clear:
```js
const isRepoPath = _folderPath ? isRepositoryPath(_folderPath) : false;
```

Or even, better, imo:
```js
const isRepoPath = !!_folderPath && isRepositoryPath(_folderPath);
```